src/coreclr/jit/gentree.cpp

```diff
@@ -21967,3 +21967,23 @@ bool GenTree::IsInvariant() const
     GenTree* lclVarTree = nullptr;
     return OperIsConst() || Compiler::impIsAddressInLocal(this, &lclVarTree);
 }
+
+//------------------------------------------------------------------------
+// IsNeverNegative: returns true if the given tree is known to be never
+//      negative such as: casts to unsiged, UDIV, UMOD, Array.Length
+//
+// Arguments:
+//    comp - Compiler object, needed for IntegralRange::ForNode
+//
+// Return Value:
+//      true if the given tree is known to be never negative
```

Comment:
```suggestion
//      True iff the given tree is known to never be negative.
```

Reply:
typo

Reply:
(I do note that `iff` is actually a [legitimate expression](https://en.wikipedia.org/wiki/If_and_only_if))

Reply:
let's leave as is, I'm not a big fan of acronyms one has to google 🙂